modules/3d/src/pointcloud/io_base.hpp

```diff
@@ -18,18 +18,24 @@ class BasePointCloudEncoder;
 using PointCloudDecoder = std::unique_ptr<BasePointCloudDecoder>;
 using PointCloudEncoder = std::unique_ptr<BasePointCloudEncoder>;
 
+// for OBJ files: to read vertices, normals and texture coords as they are given in the file
+// or duplicate them according to faces
+const int READ_AS_IS_FLAG = 1;
```

Comment:
I propose to add flag parameter to public API as it's done in `imread` and make it public option.